docs/content/about/releases.mdx

```diff
@@ -44,13 +44,13 @@ While technically the 0.y.z phase of Semantic Versioning is "anything goes", we
 
 ---
 
-## Python Version Support
+## Python version support
 
-Each Dagster release strives to support the currently active versions of python.
+Each Dagster release strives to support the currently active versions of Python.
 
-When a new version of python is released, Dagster will work to add support once Dagster's own core dependencies have been updated to support it.
+When a new version of Python is released, Dagster will work to add support once Dagster's own core dependencies have been updated to support it. **Note**: Some external libraries may not always be compatible with the latest version of Python.
```

Comment:
If we have a citable example, I think it'd be helpful. Ex: DuckDB was mentioned, but their docs only say they require Python 3.7 and newer.

Reply:
I don't have a durable example unfortunately - i suspect the next release of duckdb will have 3.12 support